packages/plugins/rubygem-foreman_discovery/rubygem-foreman_discovery.spec

```diff
@@ -33,8 +33,8 @@ Provides: foreman-plugin-%{plugin_name} = %{version}
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 8.4.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 9.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 6.0.0
+BuildRequires: %{?scl_prefix}npm(@theforeman/builder) < 7.0.0
```

Comment:
The update script keeps changing this, can I fix the script or is this expected?

Reply:
fix https://github.com/theforeman/foreman_discovery/blob/6cee7c3043fe29d88172f8c8ac99e309e1ab5c14/package.json#L28? :)